content/doc/developer/handling-requests/deprecated-reflective-access.adoc

```diff
@@ -0,0 +1,19 @@
+---
+title: Deprecated reflective access
+layout: developersection
+---
+
+Before Jenkins 2.272 Stapler would access all fields ignoring visibility restrictions.
+
+Stapler now tries to access a field first and only if it gets access denied then will it force access via 
+javadoc:java.lang.reflect.AccessibleObject#setAccessible-boolean-[setAccessible(true)].
+
+This behaviour is not recommended anymore, and will trigger IllegalReflectiveAccess warnings when running Jenkins
+on Java 11.
```

Comment:
Developer docs are generally sentence-per-line formatted.

Reply:
Well any AsciiDoc yes.

Reply:
is that what you meant? it's only one sentence.